### PR TITLE
Change email token page

### PR DESCRIPTION
**Pull request by Calum-Campbell**
Related PR's:
- Identity: https://github.com/guardian/identity/pull/1241
- Frontend : https://github.com/guardian/frontend/pull/19888

- Page for forwarding change email tokens to the API
Success : 
image

Failure : 
image

### CODE REVIEW

File: public/change-email-page.hbs [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{{# partial "pageTitle" }}{{ ChangeEmailSuccessfulText.pageTitle }}{{/ partial }}
+
+{{# partial "content" }}
+
+  {{# unless errors}}
+    <h1 class="page-heading">{{ ChangeEmailSuccessfulText.title }}</h1>
```
Comment (walaura) [inline public/change-email-page.hbs]: can you use the [new CSS thingies](https://github.com/guardian/identity-frontend/blob/5787c2229025ebc65c7317a02dd6a0d07aac47f0/public/third-party-ts-and-cs-page.hbs)? namely `layout-`.

Comment (Calum-Campbell) [reply public/change-email-page.hbs]: Updated

File: public/change-email-page.hbs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      </section>
+    {{/unless}}
+
+    <section class="layout-header">
+      {{#if errors}}
+        <h1 class="layout-header__title">{{ ChangeEmailSuccessfulText.unexpectedTitle }}</h1>
+      {{/if}}
+
+      {{#each errors}}
+        <p id="{{id}}" class="form__error">{{message}}</p>
```
Comment (walaura) [inline public/change-email-page.hbs]: `form-feedback.form-feedback--error.form-field-wrap`

File: app/com/gu/identity/frontend/models/text/ChangeEmailText.scala [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+                                               button2: String,
+                                               error:String)
+
+object ChangeEmailText {
+  def apply()(implicit messages: Messages): ChangeEmailText =
+    ChangeEmailText(
+      pageTitle = messages("changeEmailSuccessful.pageTitle"),
+      title = messages("changeEmailSuccessful.title"),
+      button1 = messages("changeEmailSuccessful.button1"),
+      button2 = messages("changeEmailSuccessful.button2"),
```
Comment (walaura) [inline app/com/gu/identity/frontend/models/text/ChangeEmailText.scala]: is there a reason these strings can't be more specific? something like `returnToGuardianAction` and `backToAccountAction` would make it clearer to see what they refer to

Comment (Calum-Campbell) [reply app/com/gu/identity/frontend/models/text/ChangeEmailText.scala]: Yep good shout, updated

**Pull request merged**